include/odp/api/spec/crypto.h

```diff
@@ -1140,9 +1144,13 @@ int odp_crypto_session_create(const odp_crypto_session_param_t *param,
  */
 int odp_crypto_session_destroy(odp_crypto_session_t session);
 
+#if ODP_DEPRECATED_API
+
 /**
  * Return crypto completion handle that is associated with event
  *
+ * @deprecated
```

Comment:
Doxygen documentation looks nicer when some text follows the "@deprecated" tag.

Reply:
Added text in v7.